stories/module-search-console-components.stories.js

```diff
@@ -66,3 +69,21 @@ generateReportBasedWidgetStories( {
 	options: pageDashboardImpressionsArgs,
 	component: DashboardImpressionsWidget,
 } );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'search-console',
+	datastore: STORE_NAME,
+	group: 'Search Console Module/Components/Dashboard/Popular Keywords Widget',
+	data: dashboardPopularKeyWordsWidgetData,
+	options: dashboardPopularKeyWordsWidgetArgs,
+	component: DashboardPopularKeywordsWidget,
+} );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'search-console',
+	datastore: STORE_NAME,
+	group: 'Search Console Module/Components/Page Dashboard/Popular Keywords Widget',
+	data: dashboardPopularKeyWordsWidgetData,
+	options: dashboardPopularKeyWordsWidgetArgs,
```

Comment:
This should use different data, which is more applicable to the page dashboard (e.g. `options` should include `url`).

Reply:
@felixarntz I updated this. Just a note here that the data returned is the same shape both with and without the `url` parameter. I have created a `page` version of the fixture to keep things consistent - even though it is the same as the non-page version.